operator/test/test.sh

```diff
@@ -149,17 +149,27 @@ test_pipelines() {
   # change ubuntu image to ubi to avoid dockerhub registry pull limit
   PIPELINE_RUN=$(curl --fail --silent "$BASE_URL/examples/v1beta1/$manifest" | sed 's|ubuntu|registry.access.redhat.com/ubi8/ubi-minimal:latest|' | sed '/serviceAccountName/d' | KUBECONFIG="$KUBECONFIG_KCP" kubectl create -f -)
   echo "$PIPELINE_RUN"
-  
-  KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=60s >/dev/null
+
+  # KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=60s >/dev/null
+  local wait_period=0
+  while [[ "$(KUBECONFIG="$KUBECONFIG_KCP" kubectl get PipelineRun -o json  | jq -r '.items[0].status.conditions[0].reason')" != "Succeeded" ]]; do
+    KUBECONFIG="$KUBECONFIG_KCP" kubectl get PipelineRun -o json ## for debugging
+    if [ "$wait_period" -gt 360 ]; then
+      echo "[ERROR]Failed to execute Pipelinerun"
```

Comment:
Redirect to stderr.

Reply:
Same as before, this is for debugging, it's removed.